coral-component-list/src/scripts/SelectListItem.js

```diff
@@ -221,7 +221,7 @@ const SelectListItem = Decorator(class extends BaseComponent(HTMLElement) {
     this.classList.add(CLASSNAME);
 
     if (!this.hasAttribute('role')) {
-      this.setAttribute('role', 'option');
+      this.setAttribute('role', 'listitem');
```

Comment:
This change should not be made to the SelectListItem control. The WAI-ARIA `option` role is a widget role, that will keep a screen reader in forms mode, allowing the user to navigate using the arrow keys. A `listitem` is a static role defining content structure. Think about this as the difference between an `<option>` element and a `li` in HTML.